src/System.Diagnostics.Tracing.Telemetry/src/System/Diagnostics/Tracing/TelemetrySource.cs

```diff
@@ -31,6 +34,8 @@ public abstract class TelemetrySource
         /// that listeners will use string prefixing to filter groups, thus having hierarchy in component 
         /// names is good.  
         /// </summary>
+        [Obsolete("Renamed to System.Diagnostics.DiagnosticSource.Write")]
+
```

Comment:
Nit: the blank lines between the attributes and the methods aren't necessary, but as these files are going to be deleted soon, doesn't really matter :smile: